readme.md

```diff
@@ -396,6 +396,7 @@
 - [PlayCanvas](https://github.com/playcanvas/awesome-playcanvas#readme) - Game engine.
 - [Game Remakes](https://github.com/radek-sprta/awesome-game-remakes#readme) - Actively maintained open-source game remakes.
 - [Flame](https://github.com/flame-engine/awesome-flame#readme) - Game engine for Flutter.
+- [Discord](https://github.com/mhxion/awesome-discord-communities#readme) - Chat with friends and communities.
```

Comment:
The title here is still not correct. Please ensure everything is in sync.

Reply:
I wasn't sure at first whether I should've added the `Communities` suffix. Thanks, done.